mkosi/__main__.py

```diff
@@ -43,6 +44,9 @@ def propagate_failed_return() -> Iterator[None]:
 @propagate_failed_return()
 def main() -> None:
     log_setup()
+    # Ensure that the name and home of the user we are running as are resolved as early as possible.
+    INVOKING_USER.name()
+    INVOKING_USER.home()
```

Comment:
Can we put these in an init() method or so and call that instead?

Reply:
Like this?